content/en/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -19,30 +19,552 @@ admission controller. This can be done effectively using a combination of dry-ru
+      <td>Mutating (MustRunAs) & Validating</td>
+      <td>
+        <i>No opinion</i>
+      </td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>supplementalGroups</code></td>
+      <td>Mutating & Validating</td>
+      <td>
+        <i>No opinion</i>
+      </td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>fsGroup</code></td>
+      <td>Mutating & Validating</td>
+      <td>
+        <i>No opinion</i>
+      </td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>readOnlyRootFilesystem</code></td>
+      <td>Mutating & Validating</td>
+      <td>
+        <i>No opinion</i>
+      </td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>defaultAllowPrivilegeEscalation</code></td>
+      <td>Mutating</td>
+      <td>
+        <i>No opinion (non-validating)</i>
+      </td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>allowPrivilegeEscalation</code></td>
+      <td>Mutating & Validating</td>
+      <td>
+        <p><i>Only mutating if set to <code>false</code></i></p>
+        <p><b>Baseline</b>: No opinion</p>
+        <p><b>Restricted</b>: <code>false</code></p>
+      </td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>allowedHostPaths</code></td>
+      <td>Validating</td>
+      <td><i>No opinion (volumes takes precedence)</i></td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>allowedFlexVolumes</code></td>
+      <td>Validating</td>
+      <td><i>No opinion (volumes takes precedence)</i></td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>allowedCSIDrivers</code></td>
+      <td>Validating</td>
+      <td><i>No opinion (volumes takes precedence)</i></td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>allowedUnsafeSysctls</code></td>
+      <td>Validating</td>
+      <td><b>Baseline & Restricted</b>: undefined / nil / empty</td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>forbiddenSysctls</code></td>
+      <td>Validating</td>
+      <td><i>No opinion</i></td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>allowedProcMountTypes</code><br><i>(alpha feature)</i></td>
+      <td>Validating</td>
+      <td><b>Baseline & Restricted</b>: <code>["Default"]</code> OR undefined / nil / empty</td>
+    </tr>
+    <tr><td colspan="3"><code>runtimeClass</code></td></tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>&nbsp;.defaultRuntimeClassName</code></td>
+      <td>Mutating</td>
+      <td><i>No opinion</i></td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap;"><code>&nbsp;.allowedRuntimeClassNames</code></td>
+      <td>Validating</td>
+      <td><i>No opinion</i></td>
+    </tr>
+    <tr><td colspan="3"><b><code>metadata.annotations</code></b></td></tr>
+    <tr>
+      <td colspan="3"><code>seccomp.security.alpha.kubernetes.io</code></td>
```

Comment:
Yeah. I did that because otherwise the column gets too wide and messes with the rest of the table formatting... I could cram it into each cell with a line break, would that be more clear? Can I leave the metadata.annototions bit on its own line?

The other option is to put it in a separate table to allow the columns to be sized differently.

Reply:
If you want the table to have a minimum width, we're open to setting a `class` on the table, or the row and having some accompanying CSS changes (ideally, in a separate commit).

I'd also prefer to have the white space style nowrap done with a class and some CSS.